project.clj

```diff
@@ -5,7 +5,8 @@
             :url "http://opensource.org/licenses/MIT"}
   :dependencies [[org.clojure/clojure "1.9.0"]
                  [weavejester/dependency "0.2.1"]]
-  :profiles {:dev {:dependencies [[org.clojure/clojurescript "1.10.597"]]}}
+  :profiles {:dev {:dependencies [[org.clojure/clojurescript "1.10.597"]]}
+             :provided {:dependencies [[org.clojure/tools.reader "1.3.2"]]}}
```

Comment:
Let's make both ClojureScript and reader dependencies provided. To be consistent we want to have them both dev or both provided.

Reply:
In that case, making them both provided would probably be the best choice I think.